c/zss.c

```diff
@@ -795,6 +795,17 @@ static int validateConfigPermissionsInner(const char *path) {
   return 0;
 }
 
+#ifdef ZSS_IGNORE_PERMISSION_PROBLEMS
+
+static int validateFilePermissions(const char *filePath) {
```

Comment:
The rest of the changes are OK, but can you please explain this? When/why is this needed? You just need permissions max 750 on the config dir and max 640 on the files in the config dir when running for the original check to pass. That's ../deploy/instance/ZLUX/serverConfig for the standard setup.

Reply:
Yeah, exactly, that was very annoying during development that I have to do that sort of thing with my sandbox. I often have to test in nonstandard environments (e.g. upload a minimal set of file to a system like SS03) and that all adds a level of headache. I'm going to do this anyway even if this is never committed.